test/spawn.jl

```diff
@@ -20,8 +20,25 @@ shcmd = `sh`
 sleepcmd = `sleep`
 lscmd = `ls`
 havebb = false
+
+function _tryonce_download_from_cache(desired_url::AbstractString)
+    cache_url = "https://cache.julialang.org/$(desired_url)"
+    cache_head_response = Downloads.request(cache_url; method = "HEAD")
```

Comment:
Why send a HEAD request if you're just going to try and download anyway?  I think it would be better to just download directly and catch the download error.

Reply:
Done